coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -82,6 +82,10 @@
         'args': '--new',
         'gui': True
     },
+    'notepadqq': {
+        'file_arg_template': '{filename}',
+        'gui': True
```

Comment:
Hey @guilherme-p , you didn't include any `args` property. Are you sure that **notepadqq** will not have any argument?

Reply:
Waiting for feedback from @palash25 who created the issue

Reply:
ping @palash25